app/frontend/src/uploadDocuments.js

```diff
@@ -1,8 +1,22 @@
-// This triggers a file input when the button is clicked
-
 document.addEventListener('DOMContentLoaded', function() {
-  document.getElementById('file-upload').addEventListener("click", function(e) {
+  inputFileUpload = document.getElementById('documents-form-documents-field')
+    ? document.getElementById('documents-form-documents-field')
+    : document.getElementById('documents-form-documents-field-error')
+
+  selectFileButton = document.getElementsByClassName('govuk-button--secondary')[0];
```

Comment:
These two should also have explicit variable declarations, please. Also, both are going to raise exceptions if the element isn't in the dom as expected.